src/DependencyInjection/CompilerPass/HandleTagsPass.php

```diff
@@ -43,38 +40,42 @@ public function process(ContainerBuilder $container)
     }
 
     /**
-     * Creates a new Context definition
+     * Creates a new Context definition.
      *
      * @param string $contextName
      */
     protected function defineContext($contextName)
     {
         $definition = new Definition('Symbid\Chainlink\Context');
-        $this->container->setDefinition(self::SERVICE_PREFIX . $contextName, $definition);
+        $this->container->setDefinition(self::SERVICE_PREFIX.$contextName, $definition);
```

Comment:
why remove the whitespace? makes it harder to notice the concatenation operator.
Also i did not find anything about it in PSR-2